### PR TITLE
Add authentication to the try now page

### PR DESCRIPTION
**Pull request by sgraband**
Moved components into separate files.
Added header that contains login/logout buttons.
User email will be displayed if logged in.
Added hint for why the login is needed.

Contributed on behalf of STMicroelectronics

Additionally Github integration was added to the keycloak.

To test:
- Enable the keycloak
- Point `keycloak.authUrl` to the keycloak (https://keycloak.34.141.62.32.nip.io/) (`/auth` is no longer needed)
- Build and use the try-now-page image

Note:
I think we should discuss, how we want to proceed with email login. It is currently still shown on the keycloak login page, but we do not support any registration process. This might be confusing.
So we should either:
- Remove the email login from the keycloak (requires custom theme)
- or add a registration process (might not be used by a lot of people)

### CODE REVIEW

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
Hi @sgraband , the changes generally look good to me and logging in via Github worked. However, after the first login, keycloak requested additional information (see screenshot). As we neither need nor want to store this information, can we get rid of that?
image

File: node/try-now-page/src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+              {app.appName}
+            </option>
+          ))}
+        </select>
+      </p>
+    )}
+    <p>
+      Powered by{' '}
+      <a href='http://theia-cloud.io' target='_blank' rel='noreferrer'>
+        Theia.Cloud
```
[the remark this answers is not in the document]
Comment (sgraband) [reply node/try-now-page/src/components/Footer.tsx]: I fixed this, but created #146 to fix the other occurences of Theia.Cloud.

**Comment by sgraband** [comment]
Thank you for the review @lucas-koehler! Unfortunately i cannot reproduce the issue, that i get asked about more details. I cleaned everything (my user in the keycloak and my github oauth session) but when i login again everything is set automatically.
Could it be that there is a problem with the email address that you used, so that the first and last name are not detected?

**Review by lucas-koehler — APPROVED** [review]
@sgraband Thanks for the updates! I checked the account I used (a dummy account) and it indeed did not have a name set. So it makes sense to request that when it expects the profile.
Of course it would be better if a name were not required but I think it is not possible to require even less than the current scope. So this is fine for me :)

**Pull request merged**